### DISCUSSION TITLE
Statamic install on Laravel app/shared hosting, error

### DISCUSSION THREAD

**Discussion by jasonday** [statamic/cms#10895] [Troubleshooting]
### Bug description

Following steps here: https://statamic.dev/installing/laravel
Laravel v11.24.1

Any command I run in the terminal, I get the below error - 
```
  Error

  Class "Statamic\Fieldtypes\Terms" not found

  at vendor/statamic/cms/src/Providers/ExtensionServiceProvider.php:327
    323▕
    324▕     protected function registerCoreExtensions($extensions)
    325▕     {
    326▕         foreach ($extensions as $extension) {
  ➜ 327▕             $extension::register();
    328▕         }
    329▕     }
    330▕
    331▕     protected function registerAppExtensions($folder, $requiredClass)

      +12 vendor frames

  13  artisan:13
      Illuminate\Foundation\Application::handleCommand()

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

### How to reproduce

Fresh install, Laravel (namecheap shared hosting)

### Environment

```yaml
Latest version
```

### Installation

Existing Laravel app

### Additional details

Fresh Laravel install

**Comment by duncanmcclean** [comment]
Does that file actually exist? It should be `vendor/statamic/cms/src/Fieldtypes/Terms.php`

We don't typically recommend hosting Statamic on shared hosting.

**Reply by dsasko** [reply]
I get the same issue with my shared hosting provider. Will get back with more info after they check their logs and see what caused the file to be deleted.

Freshly installed Statamic doesn't have this issue but when I check after a day or two I see the file no longer exists. I don't think Statamic itself is somehow responsible for deletion but rather some kind of false-positive on an antivirus check or something like that.

I'm wondering, why is shared hosting typically not recommended? Is it because of the requirements Laravel and Statamic have like certain php functions that tend to be disabled by default? Or maybe because of the amount of memory required to crunch the app? Or just the general notion of shared hosting providers doing stuff behind the curtains that may end up, for example, deleting files that host critical classes for the CMS to function?

**Reply by edalzell** [reply]
All of the above. The fact you're having random files get deleted...

**Reply by dsasko** [reply]
Just for the record, I have been told some files have been deleted via `ClamAV` or rather the `clamscan` job which is what my hosting provider uses to keep viruses away.

**Reply by jasonvarga** [reply]
If files are getting deleted by your antivirus then I think you've found the issue. You'll need to contact your host about it. Terms.php definitely doesn't have a virus. https://github.com/statamic/cms/blob/8439c2c11cd0963c8dc13b019a39112683617321/src/Fieldtypes/Terms.php

**Reply by dsasko** [reply]
After a couple of tries we finally got this resolved.
Seems like the file has been flagged by multiple AVs as hamfull. So `ClamAV` and `eXploit Scanner` from `ConfigServer` were causing the file to be deleted. 

My host has whitelisted the file for my case alone but i'm guessing further efforts should be made toward those AV companies if it were in your/our/someones interest to broaden the scale of deployment possibilities.
